src/display/api.js

```diff
@@ -405,13 +405,12 @@ var PDFDocumentLoadingTask = (function PDFDocumentLoadingTaskClosure() {
 /**
  * Abstract class to support range requests file loading.
  * @class
+ * @alias PDFJS.PDFDataRangeTransport
+ * @param {number} length
+ * @param {Uint8Array} initialData
  */
-var PDFDataRangeTransport = (function pdfDataRangeTransportClosure() {
-  /**
-   * @constructs PDFDataRangeTransport
-   * @param {number} length
-   * @param {Uint8Array} initialData
-   */
+var PDFDataRangeTransport = PDFJS.PDFDataRangeTransport =
+(function pdfDataRangeTransportClosure() {
```

Comment:
nit: wrong indenture,  break after `(function` with four spaces indent here 
(or just revert the change if possible)